personalsite/src/App.js

```diff
@@ -0,0 +1,58 @@
+import './App.css';
+// import penguin from './images/penguin.gif';
+import Navbar from './components/navbar/Navbar';
```

Comment:
you could add more functionality to the navbar by splitting some of what you have going on in `App.js` into components and then conditionally rendering the page like we did in [this example](https://github.com/overlake333/debug).